rubygem-smart_proxy_monitoring/rubygem-smart_proxy_monitoring.spec

```diff
@@ -0,0 +1,104 @@
+%global gem_name smart_proxy_monitoring
+%global plugin_name monitoring
+
+%global foreman_proxy_dir %{_datarootdir}/foreman-proxy
+%global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
+%global foreman_proxy_settingsd_dir %{_sysconfdir}/foreman-proxy/settings.d
+
+Name: rubygem-%{gem_name}
+Version: 0.0.1
+Release: 1%{?foremandist}%{?dist}
+Summary: Monitoring smart proxy plugin
+Group: Applications/Internet
+License: GPLv3
+URL: http://github.com/theforeman/smart_proxy_monitoring
+Source0: %{gem_name}-%{version}.gem
```

Comment:
This should be `http://rubygems.org/downloads/%{gem_name}-%{version}.gem` for the full download location.